app/dashboard/views.py

```diff
@@ -555,6 +555,23 @@ def unrated_bounties(request):
         'unrated': unrated_count,
     }, status=200)
 
+@csrf_exempt
+@require_POST
+def remove_portfolio(request):
+    portfolio_id = int(request.POST.get('portfolio_id'))
+    handle = request.user.profile.handle
+
+    try:
+        profile = profile_helper(handle, True)
```

Comment:
here we already have the profile from `request.user.profile` - no need to use the profile_helper

Reply:
Also, the profile isn't actually being used for anything... but we SHOULD use it to ensure the portfolio item is actually owned by this user.

Reply:
@iamonuwa please see my second comment here regarding permissions

Reply:
Gotcha!

Reply:
Fixed, please take a look

Reply:
This doesn't look right - why would the user profile be equal to a PortfolioItem?

Reply:
Fixing this now